app/handlers/courses_join.rb

```
@@ -1,7 +1,6 @@
 class CoursesJoin
   lev_handler
 
-  uses_routine AddUserAsCourseTeacher, as: :add_teacher
```

I imagine maybe you changed the style b/c you didn't want errors to bubble up.  You can just ignore the exception (see the lev readme):

```ruby
uses_routine AddUserAsCourseTeacher, as: :add_teacher,
             translations: { outputs: { type: :verbatim } },
             ignored_errors: [:user_is_already_teacher_of_course]
```